ipoib/ipoib_ci_test.sh

```diff
@@ -154,6 +97,10 @@ function test_pods_connectivity {
     /usr/local/bin/kubectl exec -i ${POD_NAME_2} ls /dev/infiniband/
     let status=status+$?
 
```

Comment:
Why do we need screen here?

Reply:
since we can only use one session in jenkins, and using `&` with `kubectl exec` was not possible when i tried last time. Screen is used to run the server, and the jenkins session is used for the client.